api/src/main/java/org/pmiops/workbench/google/DirectoryServiceImpl.java

```diff
@@ -54,19 +57,19 @@
 
   private final Provider<GoogleCredential> googleCredentialProvider;
   private final Provider<WorkbenchConfig> configProvider;
-  private final Provider<MailService> mailServiceProvider;
   private final HttpTransport httpTransport;
   private final GoogleRetryHandler retryHandler;
+  private final MandrillService mandrillService;
 
   @Autowired
   public DirectoryServiceImpl(Provider<GoogleCredential> googleCredentialProvider,
-      Provider<WorkbenchConfig> configProvider, Provider<MailService> mailServiceProvider,
+      Provider<WorkbenchConfig> configProvider, MandrillService mandrillService,
```

Comment:
I prefer the `Provider` approach to service injection, but will defer to others on this point.